concepts/05 UI Components/Stepper/30 Implement a Wizard/00 Implement a Wizard.md

```diff
@@ -0,0 +1,16 @@
+This tutorial demonstrates one of the approaches of creating wizard-like applications with Stepper. The created application integrates the [Form](/Documentation/Guide/UI_Components/Form/Overview/) and [MultiView](/Documentation/Guide/UI_Components/MultiView/Overview/) components.
```

Comment:
Generally, you don't want to write a sentence that "of" repeated close to each other. 

```suggestion
This tutorial uses a Stepper widget to create a wizard-like application. The created application integrates the [Form](/Documentation/Guide/UI_Components/Form/Overview/) and [MultiView](/Documentation/Guide/UI_Components/MultiView/Overview/) components.
```